store/models.py

```diff
@@ -2,12 +2,16 @@
 
 
 class Promotion(models.Model):
+    # TODO: Are you sure that the max length is 255, or it would be less?
```

Comment:
Well in this case it's a many to many relationship so product can apply to different promotions and promotion can apply to different products within that promotion.  So it would be the products in the promotion